packages/grid/_modules_/grid/models/colDef/gridColDef.ts

```diff
@@ -42,6 +42,10 @@ export interface GridColDef {
    * If set, it indicates that a column has fluid width. Range [0, ∞).
    */
   flex?: number;
+  /**
+   * If set, it sets the minimum width a column can have. Commonly used in fluid columns.
+   */
```

Comment:
The default is only for resizing, its a local logic.

Reply:
Oh ok. Should we consider having a default too for the minWidth? It doesn't have to be the same as for resizing. I wonder.

Reply:
I was thinking about it and if we do and a developer sets a lower `width` in the `GridColDef` then we will force them to also provide `minWidth`. I don't think it will bring much value to set a default `minWidth`.

Reply:
>  I don't think it will bring much value to set a default minWidth.

To me, the use case is identical to having a min-width on the column resizing logic. I believe the min width we have on resizing is meant to not have any columns that completely shrink up to a point where the end-users can't interact with it.

Reply:
Updated - added a default `minWidth` of 50 and removed the constant from the `useColumnResize`. Added also a new `Minimum width` section to the docs.

Reply:
Don't forget to update the default value in the JSDoc comment.